### DISCUSSION TITLE
Ability to add & save custom word sets

### DISCUSSION THREAD

**Discussion by MichaelKim39** [monkeytypegame/monkeytype#1001] [Ideas] [answered]
Similar to the system of practicing missed words after a test, simply add a list of words you would like to practice and be able to generate a long list of those words in random order.

Would address alot of desired functionality like left hand / right hand mode, 500 word english language set, practicing word sets from other websites. 

If language support for things like korean is a concern, you can limit to english characters, punctuation and numbers with a filter initally.

**Comment by Miodec** [comment, marked as answer]
Custom mode does exactly that

**Reply by MichaelKim39** [reply]
Wow, I just realised the random option does what I suggested, sorry. But I still think that having the option to save separate lists of words for your account and easily swap between them can be useful.

**Reply by Miodec** [reply]
It is planned, but very low on the priority list.
